### ISSUE TITLE
No error raised when keyword-only argument is missing in class statement

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#144]
Originally reported by: **Antony Lee (BitBucket: [anntzer](http://bitbucket.org/anntzer), GitHub: @anntzer?)**

---

Certainly somewhat an edge case but...

```
#!python
class M(type):
    def __new__(mcs, name, bases, ns, *, foo):
        return super().__new__(mcs, name, bases, ns)

class C(metaclass=M):
    pass
```

This should trigger an error as the keyword-only argument `foo` is missing in the class statement (after all the metaclass' `__new__` can be statically determined here).

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/144

**Comment by degustaf** [comment]
I started working on this.  It does not seem like astroid collects any custom args or kwargs that are passed to the class definition.
